Fabric.Terminology.SqlServer/Persistence/SqlValueSetBackingItemRepository.cs

```diff
@@ -95,31 +95,39 @@ public IReadOnlyCollection<IValueSetBackingItem> GetValueSetBackingItems(IEnumer
 
             backingItems.AddRange(
                 this.QueryValueSetBackingItems(remaining, codeSystemGuids.ToList())
-                    .Select(bi => this.cacheManager.GetOrSet(bi.ValueSetGuid, () => bi)).Values());
+                    .Select(bi => this.cacheManager.GetOrSet(bi.ValueSetGuid, () => bi))
```

Comment:
You can use the LINQ `SelectMany` to avoid needing to call `.Values()` afterward.